vars/protecodeExecuteScan.groovy

```diff
@@ -72,3 +67,14 @@ void call(Map parameters = [:]) {
     }
 }
 
+String getParametersJSON(Map parameters = [:]){
```

Comment:
The same is done in piperExecuteBin. Can we get rid of the code duplication?

Reply:
But maybe can also be done when trying to also use piperExecuteBin for this step. So I am fine with the current state.